### DISCUSSION TITLE
X18 Storage err

### DISCUSSION THREAD

**Discussion by Asafsol** [FrSkyRC/ETHOS-Feedback-Community#4527] [Q&A]
During setup, the screen went dark and the main button light became blue. nothing worked.
the back was pretty hot
I disconnected the battery for few moments and reconnected.
Then the ethos system said: storage err and reset to factory settings and data.

WHY?
scary if it will happen in mid flight

edit:
now the restore file simply doesnt work
for the past 40 mins its just "working" but nothing happen

**Comment by bsongis-frsky** [comment]
Are you sure that your battery was charged enough?

**Comment by Asafsol** [comment]
Yes it is

Brg
Asaf Sol


בתאריך יום ג׳, 17 בספט׳ 2024 ב-19:48 מאת Bertrand Songis @ FrSky <
***@***.***>:
